ntoskrnl/config/cmsysini.c

```diff
@@ -138,7 +138,7 @@ CmpDeleteKeyObject(PVOID DeletedObject)
         if (Kcb)
         {
             /* Delist the key */
-            DelistKeyBodyFromKCB(KeyBody, FALSE);
+            DelistKeyBodyFromKCB(KeyBody, KeyBody->KcbLocked ? TRUE : FALSE);
```

Comment:
If the KcbLocked is already a boolean, there shouldn't be any need to do the `? TRUE : FALSE`.